lib/mappers/michigan_mapper.py

```diff
@@ -85,15 +85,26 @@ def map_date_and_publisher(self):
             "date": [],
             "publisher": []
         }
- 
+
+        date_begin, date_end = None, None
```

Comment:
If these declarations could be moved down into scope of the `try` block, right after line 94, it would reduce the possibility of confusion over how they're used. They're just temporary variables that are used for constructing the properties of `_dict` below.